src/Spryker/Zed/RabbitMq/RabbitMqConfig.php

```diff
@@ -100,4 +103,12 @@ protected function getApiPort()
     {
         return $this->get(RabbitMqEnv::RABBITMQ_API_PORT);
     }
+
+     /**
+     * @return string
+     */
+    protected function getApiScheme()
+    {
+        return $this->get(RabbitMqEnv::RABBITMQ_SCHEME);
```

Comment:
Please provide `http` as a default value otherwise is not backward compatible
`return $this->get(RabbitMqEnv::RABBITMQ_SCHEME, 'http');`

Reply:
please also run CS, travis is red